src/msg/async/AsyncMessenger.h

```diff
@@ -376,6 +313,7 @@ class AsyncMessenger : public SimplePolicyMessenger {
 
 public:
 
+  std::shared_ptr<NetworkStack> stack;
```

Comment:
why is `stack` public?  and iiuc, `stack` is not owned by `AsyncMessenger`, it is held by `CephContext`. so we should not use `std::shared_ptr`, right?